bisect_ppx-ocamlbuild.opam

```diff
@@ -0,0 +1,24 @@
+name: "bisect_ppx-ocamlbuild"
```

Comment:
I guess breaking upper bounds is a problem in some cases. I usually want them to break though, as I'm trying to install a dev version "no matter what," so I assumed this was deliberate.